bndtools.core/src/bndtools/central/Central.java

```diff
@@ -461,6 +465,9 @@ public static void refreshPlugins() throws Exception {
                         l.modelChanged(p);
                 }
 
+                for (RepositoryPlugin rp : changedRepos) {
+                    repositoriesViewRefresher.repositoryRefreshed(rp);
+                }
             } catch (Exception e) {
                 e.printStackTrace();
                 throw new RuntimeException(e);
```

Comment:
What about `refreshPlugin(Refreshable plugin)` below? Shouldn't is also call `repositoriesViewRefresher.repositoryRefreshed(rp)` if the plugin is a RepositoryPlugin?